plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -1208,41 +1208,6 @@ public function testEditConflict()
         $this->assertContentType('application/vnd.api+json');
     }
 
-    /**
-     * Test edit method with invalid data.
-     *
-     * @return void
-     *
-     * @covers ::resource()
-     * @covers ::initialize()
-     */
-    public function testEditInvalid()
```

Comment:
I'd like to keep this test - maybe with another case causing a `400` response (like a user with an existing `email`?) - without usgin `uname`

Also keep the 404 response on PATCH, even if trivial